config/FpConfig.fpp

```diff
@@ -1,16 +1,85 @@
-type FwChanIdType
-type FwDpIdType
-type FwDpPriorityType
-type FwEnumStoreType
-type FwEventIdType
-type FwIndexType
-type FwOpcodeType
-type FwPacketDescriptorType
-type FwPrmIdType
-type FwSignedSizeType
-type FwSizeStoreType
-type FwSizeType
-type FwTimeBaseStoreType
-type FwTimeContextStoreType
-type FwTlmPacketizeIdType
-type FwTraceIdType
+# ======================================================================
+# \title  Fw/FPrimeBasicTypes.hpp
```

Comment:
I think the file name should be .fpp, not .hpp

Reply:
Good catch!  I'll fix.